languages/ruby/exercises/concept/instance-variables/.docs/introduction.md

```diff
@@ -0,0 +1,56 @@
+## Object state, instance variables
+
+Objects can hold their own state by setting _instance variables_. _Instance variables_ are created a prefixing `@` to the variable name.
+
+```ruby
+@instance_variable = 2
+```
+
+This is often done by defining an `initialize` method to set an initial state.
+
+```ruby
+class Airplane
+  def initialize
+    @wings = 2
+  end
+end
+```
+
+The `initialize` method may also take arguments, so that each instance can start with a custom state:
+
+```ruby
+class Suitcase
+  def initialize(locked)
+    @locked = locked
+  end
+end
+```
+
+Consider _instance variables_ to be private from external read and writes. _Class instance methods_ should be used for getting and setting instance variables:
+
+```ruby
+class Suitcase
+  #...
+
+  def locked? # Methods returning true or false should have trailing `?`
```

Comment:
Methods that can be used as a Boolean often have a trailing `?` as part of the name.  We call these "query methods".

This hints at the true and false, while leaving it open.  The realization of why it is stated this way becomes apparent when we talk about "truthy" and "falsey".

Reply:
Awesome, I didn't know that they were called that, will integrate the change!

Reply:
```suggestion
  def locked? # Query methods should be named with a trailing `?`
```